.github/workflows/build.yaml

```diff
@@ -59,9 +57,11 @@ jobs:
               with:
                   php-version: ${{ matrix.php }}
             - run: |
-                  sed -ri 's/"symfony\/(.+)": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
+                  sed -ri 's/"symfony\/(expression-language|framework-bundle|templating)": "(.+)"/"symfony\/\1": "'${{ matrix.symfony }}'"/' composer.json;
               if: matrix.symfony
             - run: composer config minimum-stability dev
               if: matrix.dev
             - run: composer update --no-interaction --no-progress --ansi ${{ matrix.composer_option }}
             - run: vendor/bin/phpunit
+              env:
+                  SYMFONY_DEPRECATIONS_HELPER: max[self]=1
```

Comment:
This is a no-go. Tests covering our own deprecations should use `@group legacy` instead, to make deprecations allowed in them.

Reply:
I added the needed groups to our legacy tests